### ISSUE TITLE
How to use label

### ISSUE THREAD

**Issue by Yanxudong23** [wnhsu/ScalableFHVAE#3]
hello, the code just make one sequence as a class. How can I use the variables "lab_specs" to solve it?

**Comment by wnhsu** [comment]
`lab_specs` is currently used for visualization. You can find here (https://github.com/wnhsu/ScalableFHVAE/blob/17a1d8dddb5313c72e0986d1ba21bb10aad23b91/scripts/train/train_loaders.py) that seq2idx maps from sequence name to sequence ID (i.e., class ID). You can change the code here to map sequences from the same class (e.g., speaker) to the same ID, where ID ranges in [0, nClass-1]

**Comment by Yanxudong23** [comment]
yes, I solve it on the Sunday, thanks very much~

**Comment by Yanxudong23** [comment]
hello， how can I test a new speaker without training, just put the wav file into the dev file?
